cristin-person/src/main/java/no/unit/nva/cristin/person/handler/QueryCristinPersonHandler.java

```diff
@@ -53,18 +53,19 @@ public QueryCristinPersonHandler(CristinPersonApiClient apiClient, Environment e
     }
 
     @Override
+    @SuppressWarnings({"PMD.DataflowAnomalyAnalysis"})
```

Comment:
Can we exclude this warning in pmd config file so we dont have to worry about it?